src/room/GroupCallView.tsx

```diff
@@ -246,6 +244,9 @@ export const GroupCallView: FC<Props> = ({
 
   useEffect(() => {
     if (widget && isJoined) {
+      // set widget to sticky once joined.
+      widget!.api.setAlwaysOnScreen(true);
```

Comment:
Can the other invocation of `setAlwaysOnScreen` in this file be removed now that this is here?